README.md

```diff
@@ -342,29 +368,165 @@ Your Galasa Ecosystem will now use the newly generated encryption key to encrypt
+      ```
+      127.0.0.1 example.com
+      ```
+
+3. Add an entry to minikube's CoreDNS ConfigMap so that deployed pods can resolve the external hostname internally:
+    1. Run `minikube ip` and take a note of the displayed IP address as it will be used in the following steps:
+
+        ```
+        > minikube ip
+        192.168.49.2
+        ```
+
+    1. Open the CoreDNS ConfigMap for editing by running:
+
+        ```
+        kubectl -n kube-system edit configmap coredns
+        ```
+
+    2. Add a new entry under `Corefile` as follows, replacing `192.168.49.2` with the output of `minikube ip` from step 1 and  (optionally) replacing `example.com` with your hostname value:
+
+        ```
+        example.com:53 {
+          hosts {
+            192.168.49.2 example.com
+            fallthrough
+          }
+        }
+        ```
+
+        The ConfigMap should now look like this:
+        
+        ```yaml
+        apiVersion: v1
+        data:
+          Corefile: |
+            .:53 {
+                log
+                errors
+                health {
+                  lameduck 5s
+                }
+                ready
+                kubernetes cluster.local in-addr.arpa ip6.arpa {
+                  pods insecure
+                  fallthrough in-addr.arpa ip6.arpa
+                  ttl 30
+                }
+                prometheus :9153
+                hosts {
+                  192.168.5.2 host.minikube.internal
+                  fallthrough
+                }
+                forward . /etc/resolv.conf {
+                  max_concurrent 1000
+                }
+                cache 30 {
+                  disable success cluster.local
+                  disable denial cluster.local
+                }
+                loop
+                reload
+                loadbalance
+            }
+            example.com:53 {
+                hosts {
+                    192.168.49.2 example.com
+                    fallthrough
+                }
+            }
+        kind: ConfigMap
+        metadata:
+          creationTimestamp: "2025-02-26T13:56:33Z"
+          name: coredns
+          namespace: kube-system
+          resourceVersion: "7613"
+          uid: 04f54d19-68ce-4828-8365-5a35a266a0cd
+        ```
+
+    3. Save and exit the editor
+
+    4. Restart minikube's CoreDNS deployment by running:
+        ```
+        kubectl -n kube-system rollout restart deployment coredns
+        ```
+
+4. Follow the instructions in the [Configuring Ingress](#configuring-ingress) and [Configuring Dex](#configuring-dex) sections to update the rest of your values.yaml file.
+
+5. Once you have updated your values.yaml file, run the following command, providing the path to the [`ecosystem`](./charts/ecosystem) directory in this repository (e.g. `~/helm/charts/ecosystem`).
+
+    ```console
+    helm install <release-name> /path/to/helm/charts/ecosystem
+    ```
+
```

Comment:
Perhaps provide an example successful output of what this should look like

Reply:
I think it would also be useful to state that it takes a few minutes to come up on minikube despite the helm install output coming through very quickly, so they should wait a couple of minutes before moving to helm test, or ensure to go kubectl get pods to check all are Running first

Reply:
Added example output and additional step to wait and monitor status in https://github.com/galasa-dev/helm/pull/69/commits/9eb3a19d0eca51ae2ce5583a476fd20cac27893f